public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Clinician Dashboard</title>
```

Comment:
Patient Health Data

Reply:
resolve in fd43657a6801ddb87fa3b2b74d06951f6219bc7a